api/v1/db/oracledb/LearningDB.sql

```diff
@@ -0,0 +1,192 @@
+DROP TABLE CARD_SUITS;
+DROP TABLE CARD_NUMBERS;
+DROP TABLE GAMES;
+DROP TABLE MEMBERS;
+DROP TABLE PARTICIPATIONS;
+DROP TABLE ROUNDS;
+DROP TABLE STATUSES;
```

Comment:
These statements don't drop all of the tables created in the script. Also, the order needs to be done in a way that will drop all tables with foreign key constraints before the tables being referenced. An easy way to do this is to drop all of the tables in the reverse order that they were declared.